src/components/Header.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+
+import '../styles/Header.scss';
+
+const logoDesktop = './images/logo-desktop.svg';
+const logoMobile = './images/logo-mobile.svg';
+const iconImportCSV = './images/importCSV.svg';
+const minDesktopWidth: number = 768;
+
+const Header = () => {
+    const [isResized, setIsResized] = useState(window.outerWidth >= minDesktopWidth);
```

Comment:
`isResized` according to the variable name it is used to save the state of the application when it was resized. So when we load the app it should be `false` and when we resize for the first time it will be changed to `true` until the app refresh?

Reply:
yes, you are right. The name of variable was not correct, as it makes other logic, so i decided to renamed variable.